packages/vaadin-virtual-list/test/wheel-scrolling.test.js

```diff
@@ -0,0 +1,195 @@
+import { expect } from '@esm-bundle/chai';
+import { fixtureSync, nextRender } from '@vaadin/testing-helpers';
+import Sinon from 'sinon';
+import { Virtualizer } from '../src/virtualizer.js';
+
+function canScroll(el, deltaY) {
+  return (
+    (['auto', 'scroll'].indexOf(getComputedStyle(el).overflow) !== -1 &&
+      deltaY > 0 &&
```

Comment:
I would suggest to decouple this condition into smaller ones because it uses a few logical operators that makes it quite difficult to understand.

Reply:
Done.